platformio.ini

```diff
@@ -28,6 +28,7 @@ lib_deps =
     nrf24/RF24 @ ^1.4.5
     olikraus/U8g2 @ ^2.34.13
     buelowp/sunset @ ^1.1.7
+    https://github.com/janchri/WireGuard-ESP32-Arduino

```

Comment:
ESPHome uses this in their platformio.ini:
```yaml
droscy/esp_wireguard@0.4.2            ; wireguard
```

Reply:
And that is also fork because of some esphome optimizations, but works perfectly.
Maybe it make sense to start new attempt with this "esphome" fork...